apps/namadillo/src/App/Transfer/TransferModule.tsx

```diff
@@ -23,6 +24,7 @@ export type TransferModuleConfig = {
   chain?: Chain;
   onChangeChain?: (chain: Chain) => void;
   isShielded?: boolean;
+  errorMessage?: string;
```

Comment:
It looks like this `errorMessage` is not used anywhere, just the one in `TransferModuleProps`. I think you could also remove the `export` from this type since it isn't used anywhere else.